cli/unstable_ansi.ts

```diff
@@ -0,0 +1,961 @@
+// Copyright 2018-2025 the Deno authors. MIT license.
+
+/**
+ * Ansi is a module that offers export function s and variables that returns
+ * various Ansi Escape Sequences. This class is not an exhaustive list of what
+ * is possible with Ansi Escape Sequences, nor does it guarantee that every
+ * code will work in every terminal. The only way to guarantee that only one
+ * code will work in a particular terminal, is to check for yourself. Calling
+ * these export function s and variables does not automatically change the
+ * terminal settings. Only once they are passed to stdout or stderr will they
+ * take effect.
+ *
+ * These codes were based off the
+ * [xterm reference](https://invisible-island.net/xterm/ctlseqs/ctlseqs.html).
+ *
+ * @example Basic Usage
+ * ```ts ignore
+ * import * as Ansi from "@std/cli/unstable_ansi";
+ *
+ * console.log(Ansi.doubleHeightTop + "Hello World!");
+ * console.log(Ansi.doubleHeightBottom + "Hello World!");
+ * ```
+ *
+ * @module
+ */
+
+/**
+ * Causes content on the current line to enlarge, showing only the top half
+ * of characters with each character taking up two columns. Can be used in
+ * combination with {@linkcode Ansi.doubleHeightBottom} on the next line to
```

Comment:
```suggestion
 * combination with {@linkcode doubleHeightBottom} on the next line to
```
AFAIK fixes linking. Ditto elsewhere.